ci_tools/bumpversion.sh

```diff
@@ -1,5 +1,16 @@
-bump2version patch --message "Bump version: {current_version} to {new_version} [ci skip]"
-git config --local user.email "engineering@finbourne.com"
-git config --local user.name "finbourne-bot-public"
-git push https://finbourne-bot-public:${gh_token}@github.com/finbourne/lusid-python-tools.git HEAD:$TRAVIS_BRANCH
-echo "Making commit with incremented version"
\ No newline at end of file
+#!/bin/bash
+
+last_commit_message=$(git log -1 HEAD --pretty=format:%s)
+
+echo "Last commit message: $last_commit_message"
+
+if [[ "$last_commit_message" == "Bump version"* ]]; then
+        echo "last commit is a bump version commit... skipping..."
+else
+        echo "Bumping packaging patch version by one"
```

Comment:
Maybe just a simple "Incrementing patch version to $new_version"

Reply:
I'm not sure the Travis environment will know the $new_version variable? We could always scrape this from PyPi?

Reply:
if we don't have it at hadn't then I think its ok to leave

Reply:
See latest commit. We can take the version from `__version__.py` and then increment in the shell script.